addons/website_slides/controllers/mail.py

```diff
@@ -8,7 +8,7 @@
 from odoo import http
 from odoo.http import request
 from odoo.addons.portal.controllers.mail import _check_special_access, PortalChatter
-from odoo.tools import plaintext2html
+from odoo.tools import plaintext2html,html2plaintext
```

Comment:
Space after commas that separate imports

Reply:
Fixed